index.html

```diff
@@ -66,7 +66,7 @@ <h3>Multi-Post Stories</h3>
               <li>bootstrap</li>
               <li>Ruby</li>
             </ul>
-            <button type="submit">See Project</button>
+            <button class="project-button" type="submit">See Project</button>
```

Comment:
- [ ] As requested by the previous reviewer, kindly have an array of objects to store JavaScript objects containing the information for each project.
An instance of a single project would look like this:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
After, the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript.**